db/models/reviews.models.js

```diff
@@ -11,6 +14,7 @@ exports.fetchReviewById = (id) => {
           msg: `No user found for user ${id}`,
         });
       } else {
+        review.comment_count = Number(review.comment_count);
```

Comment:
this is nicely handled - you can also achieve this in your SQL statment - the docs are here [The CAST operator in PostgreSQL](https://www.sqlservercentral.com/articles/the-cast-operator-in-postgresql#:~:text=Cast%20is%20a%20technique%20in,datetime%20datatype)